src/api/helpers/encrypt/index.ts

```diff
@@ -0,0 +1,28 @@
+import crypto from 'crypto';
+import { FINANCIAL_DETAILS } from '../../constants';
+import generateKey from './generate-key';
+import generateInitialisationVector from './generate-initialisation-vector';
+
+const { ENCRYPTION_METHOD, ENCODING, STRING_ENCODING, OUTPUT_ENCODING } = FINANCIAL_DETAILS.ENCRYPTION;
+
+const key = generateKey();
+// initialisation vector
+const iv = generateInitialisationVector();
+
+/**
+ * encrypt
+ * encrypts data
+ * returns encrypted value and initialisation vector
+ * @param dataToEncrypt
+ * @returns {Object} value and initialisation vector
+ */
+const encrypt = (dataToEncrypt: any) => {
```

Comment:
Could we create a new type and use as a return type here, would save having vague `@returns {Object}` as well

Should`dataToEncrypt` param be `any`, should it not be a string?

Reply:
I've kept it as any incase in the future we pass it just a number for example